test/test_uninstall.sh

```diff
@@ -56,7 +56,7 @@ function test_uninstall() {
   tofuenv install "${v}" || return 1;
   tofuenv uninstall "${v}" || return 1;
   log 'info' 'Confirming uninstall success; an error indicates success:';
-  check_active_version "${v}" && return 1 || return 0;
+  env TOFUENV_AUTO_INSTALL=false "${TOFUENV_ROOT}/bin/tofu" version && return 1 || return 0;
```

Comment:
I bypass the helper function here, but this guarantees that `tofu` fails if the version is not installed.